test_tracetools/src/mark_process.cpp

```diff
@@ -19,6 +19,8 @@
 #include "rcpputils/env.hpp"
 #endif  // TRACETOOLS_DISABLED
 
+#include <string>
```

Comment:
While we are doing this, should we also add:

```
#include <string_view>
```

Reply:
I was just looking into whether `<string>` includes `<string_view>`, because technically `<string>` is only needed when `#ifndef TRACETOOLS_DISABLED`.

I'll just add `<string_view>` here and move `<string>` inside the `#ifndef TRACETOOLS_DISABLED` above.

Reply:
Done!